tools/test.py

```diff
@@ -174,6 +174,21 @@ def main():
                               **cfg.data.get('test_dataloader', {}))
     data_loader = build_dataloader(dataset, **dataloader_setting)
 
+    if cfg.model.get('backbone', None):
+        # remove pretrain steps for testing
+        if hasattr(cfg.model.backbone, 'pretrained'):
+            cfg.model.backbone.pretrained = None
+        if hasattr(cfg.model.backbone, 'pretrained2d'):
```

Comment:
No need to set pretrained2d, since pretrained is None

Reply:
ur right